hack/vendor.sh

```diff
@@ -17,8 +17,8 @@ if [ $# -eq 0 ] || [ "$1" = "archive/tar" ]; then
 	: "${GO_VERSION:=$(awk -F '[ =]' '$1 == "ARG" && $2 == "GO_VERSION" { print $3; exit }' ./Dockerfile)}"
 	rm -rf vendor/archive
 	mkdir -p ./vendor/archive/tar
-	echo "downloading: https://golang.org/dl/go${GO_VERSION}.src.tar.gz"
-	curl -fsSL "https://golang.org/dl/go${GO_VERSION}.src.tar.gz" \
+	echo "downloading: https://golang.org/dl/go${GO_VERSION%.*}.src.tar.gz"
+	curl -fsSL "https://golang.org/dl/go${GO_VERSION%.*}.src.tar.gz" \
```

Comment:
We probably want this to be specific to just the trailing `.0` so we don't drop patch releases, right?

```suggestion
	echo "downloading: https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz"
	curl -fsSL "https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz" \
```

Reply:
Ah, thanks! I always struggle with these and forget the correct syntax, LOL. Yup, that's what my intent was